api/models/userDao.js

```diff
@@ -0,0 +1,40 @@
+const appDataSource = require('./dataSource')
+
+const signUp = async(kakaoUserInfo, res) => {
+    const signUpUser = await appDataSource.query(
+        `INSERT INTO 
+            users (
+                name,
+                email,
+                gender,
+                social_id
+            ) VALUES (
+                ?,
+                ?,
+                ?,
+                ?
+            )
+            `, [kakaoUserInfo.nickname, 
+                kakaoUserInfo.email, 
+                kakaoUserInfo.gender, 
+                kakaoUserInfo.sub]
+        )
+        return signUpUser
+}
```

Comment:
```suggestion
const createUser = async(kakaoUserInfo, res) => {
    return await appDataSource.query(
        `INSERT INTO 
            users (
                name,
                email,
                gender,
                social_id
            ) VALUES (
                ?,
                ?,
                ?,
                ?
            )
            `, [kakaoUserInfo.nickname, 
                kakaoUserInfo.email, 
                kakaoUserInfo.gender, 
                kakaoUserInfo.sub]
        )
}
```
dao는 CRUD에 맞춰서 naming을 해주시는게 일반적인 컨벤션입니다! (ex. getProduct, createReview)
또한 다음과 같이 꼭 변수를 선언하지 않고도 query를 할 수 있을 것 같아요 :)